_i18n/es/resources/user-guides/mine-to-pool.md

```diff
@@ -52,64 +37,57 @@ make and copy the config file:
 
-After this, your config should look something like this:
+Después de esto, tu archivo config  se deberá ver algo así:
 
 ![image3](png/mine_to_pool/3.png)
 
-# Running the miner
+# Corriendo el minero
 
-**Save the config** file and run the miner!
+**Guarda el archivo config** y corre el minero.
 
 ![image4](png/mine_to_pool/4.png)
 
-Some pools allow you to monitor your hashrate by pasting your address into their
-website. You can also monitor your hashrate by pressing the `h` key.
+Algunas pools te permiten monitorear tu hashrate al pegar tu dirección en su sitio web. También puedes monitorear tu hashrate presionando la tecla `h`.
 
-# Tuning the miner
+# Afinando el minero
 
-You might see nasty messages like this:
+Es posible que veas mensajes desagradables como este:
 
 	[2017-07-09 12:04:02] : MEMORY ALLOC FAILED: mmap failed
 
-This means that you can get around a 20% hashrate boost by enabling large pages.
+Esto significa que puedes obtener alrededor de un 20% extra de hashrate activando las páginas grandes.
 
-## Large pages on Linux
+## Páginas grandes en Linux
 
-Firstly stop the miner (if it's running), run the following commands to enable
-large pages and then start the miner as root:
+En primer lugar, detén el minero (si está corriendo), y corre los siguientes comandos para activar las páginas grandes e iniciar el minero como root:
 
 	sudo sysctl -w vm.nr_hugepages=128
 	sudo ./xmr-stak-cpu
 
-## Large pages on Windows
-
-Taken from `config.txt`:
-
->By default we will try to allocate large pages. This means you need to "Run As Administrator" on Windows
-You need to edit your system's group policies to enable locking large pages. Here are the steps from MSDN
-1. On the Start menu, click Run. In the Open box, type gpedit.msc.
-2. On the Local Group Policy Editor console, expand Computer Configuration, and then expand Windows Settings.
-3. Expand Security Settings, and then expand Local Policies.
-4. Select the User Rights Assignment folder.
-5. The policies will be displayed in the details pane.
-6. In the pane, double-click Lock pages in memory.
-7. In the Local Security Setting – Lock pages in memory dialog box, click Add User or Group.
-8. In the Select Users, Service Accounts, or Groups dialog box, add an account that you will run the miner on
-9. Reboot for change to take effect.
+## Páginas grandes en Windows
+
+Tomado de `config.txt`:
+
+>Por defecto trataremos de asignar páginas grandes. Esto significa que necesitas "Correr como Administrador" en Windows. Necesitas editar las políticas de grupo del sistema para habilitar el bloqueo de páginas grandes. Aquí están los pasos de MSDN
+1. En el menú de inicio, haz clic en Ejecutar. En la ventana abierta, escribe gpedit.msc.
+2. En la consola de Local Group Policy Editor, expande Computer Configuration, y luego expande Windows Settings.
```

Comment:
"Local Group Policy Editor = Editor de directivas de grupo local"

Check the [Microsoft Terminology Search](https://www.microsoft.com/en-us/language/Search?&searchTerm=unconfirmed&langID=Spanish&Source=true&productid=All).